### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:)

### CODE REVIEW

File: po/nl_NL.po [hunk 1]
```diff
@@ -0,0 +1,513 @@
+# French translations for oomox package
```
Comment (actionless) [inline po/nl_NL.po]: i don't think you should still keep this copyright paragraph here

File: po/nl_NL.po [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+msgstr "Ongevoelig item {id}"
+
+#: oomox_gui/preview.py:194
+msgid "Preview:"
+msgstr "Voorbeeld:"
+
+#~ msgid "Theme applied successfully"
+#~ msgstr "Thema is toegepast"
+
+#~ msgid "Spotify options"
```
Comment (actionless) [inline po/nl_NL.po]: why all those are commented out?

File: po/nl_NL.po (hunk 1 above)
Comment (Vistaus) [reply po/nl_NL.po]: You're right, I'll change that. Thanks for the heads-up!

File: po/nl_NL.po (hunk 2 above)
Comment (Vistaus) [reply po/nl_NL.po]: I dunno, you'd have to ask Poedit.

Comment (Vistaus) [reply po/nl_NL.po]: I manually fixed them now.

**Comment by actionless** [comment]
thanks a lot!

**Pull request merged**